Makefile.posix

```diff
@@ -36,10 +36,20 @@ config.jou: Makefile.posix
 	echo 'const JOU_CLANG_PATH: byte* = "$(shell command -v `$(LLVM_CONFIG) --bindir`/clang || command -v clang)"' >> config.jou
 	echo '' >> config.jou
 	echo '@public' >> config.jou
-	echo 'const LLVM_HAS_AARCH64: bool = $(shell $(LLVM_CONFIG) --components | grep -q aarch64 && echo True || echo False)' >> config.jou
+	echo 'const LLVM_HAS_AARCH64: bool = $(shell $(LLVM_CONFIG) --targets-built | grep -q AArch64 && echo True || echo False)' >> config.jou
+	echo '' >> config.jou
+	echo '@public' >> config.jou
+	echo 'const LLVM_HAS_ARM: bool = $(shell $(LLVM_CONFIG) --targets-built | grep -q ARM && echo True || echo False)' >> config.jou
+	echo '' >> config.jou
+	echo '@public' >> config.jou
+	echo 'const LLVM_HAS_X86: bool = $(shell $(LLVM_CONFIG) --targets-built | grep -q X86 && echo True || echo False)' >> config.jou
+	echo '' >> config.jou
+	echo '@public' >> config.jou
+	echo 'const JOU_TARGET: byte* = ""' >> config.jou
```

Comment:
If I understand this correctly, empty string means "use LLVM's default", but you can set this to work around the raspberry pi CPU problem? So you're going to add the "if grep raspberry pi" workaround here?

To future-proof this, a better name would be `JOU_NATIVE_TARGET`.